Neos.ContentRepository.Core/Classes/NodeType/NodeType.php

```diff
@@ -527,18 +222,30 @@ public function getNodeTypeNameOfTetheredNode(NodeName $nodeName): NodeTypeName
      * Otherwise, isNodeTypeAllowedAsChildToTetheredNode() needs to be called on the *parent node type*.
      *
      * @return boolean true if the $nodeType is allowed as child node, false otherwise.
+     * @deprecated with Neos 9.0 – {@see childNodeTypeConstraints} should be used instead
      */
     public function allowsChildNodeType(NodeType $nodeType): bool
```

Comment:
so it will be

```
$nodeType->childNodeTypeConstraints->isNodeTypeAllowed($nodeTypeNameInQuestion);
```

and not something to ask the nodetypemanager? ... seems to make sense.